test/module/irohad/validation/query_execution.cpp

```diff
@@ -647,37 +747,39 @@ TEST_F(GetSignatoriesTest, DifferentDomainAccountInValidCase) {
  * @then Return error
  */
 TEST_F(GetSignatoriesTest, NoAccountExist) {
-  get_signatories->account_id = "none";
+  auto query = TestQueryBuilder()
+                   .creatorAccountId(admin_id)
+                   .getSignatories("none")
+                   .build();
+
   role_permissions = {can_get_all_signatories};
 
   EXPECT_CALL(*wsv_query, getAccountRoles(admin_id))
       .WillOnce(Return(admin_roles));
   EXPECT_CALL(*wsv_query, getRolePermissions(admin_role))
       .WillOnce(Return(role_permissions));
 
-  EXPECT_CALL(*wsv_query, getSignatories(get_signatories->account_id))
-      .WillOnce(Return(boost::none));
+  EXPECT_CALL(*wsv_query, getSignatories("none")).WillOnce(Return(boost::none));
```

Comment:
Why getSignatorites has been reworked with "none" instead of `accountId()`.

Reply:
`account_id` was proto field, it was set in constructor and then changed in every test, and it was hard to trace all changes, so I removed it, also because we use builders now